php-frontend/src/test/java/org/sonar/php/tree/symbols/ScopeTest.java

```diff
@@ -71,6 +71,15 @@ public void function_expression_scope() throws Exception {
     assertNotNull(functionExprScope.getSymbol("$e"));
   }
 
+  @Test
+  public void arrow_function_expression_scope() throws Exception {
+    Scope scope = getScopeFor(Kind.ARROW_FUNCTION_EXPRESSION);
+
+    assertThat(scope.getSymbols(Symbol.Kind.PARAMETER)).extracting(Symbol::name).containsExactly("$a");
+    assertThat(scope.getSymbols(Symbol.Kind.VARIABLE)).isEmpty();
```

Comment:
we discussed that captured variables should be different symbols because they can be modified independently. This seems to be not the case here.

Reply:
I added a comment to explain why on https://jira.sonarsource.com/browse/SONARPHP-943